src/Components/WebAssembly/WebAssembly/src/Http/BrowserRequestCache.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Components.WebAssembly.Http
+{
+    /// <summary>
+    /// The cache mode of the request. It controls how the request will interact with the browser's HTTP cache.
+    /// </summary>
+    public enum BrowserRequestCache
+    {
+        /// <summary>
+        /// The browser looks for a matching request in its HTTP cache.
+        /// </summary>
+        Default,
+
+        /// <summary>
+        /// The browser fetches the resource from the remote server without first looking in the cache,
+        /// and will not update the cache with the downloaded resource.
+        /// </summary>
+        NoStore,
+
+        /// <summary>
+        /// The browser fetches the resource from the remote server without first looking in the cache,
+        /// but then will update the cache with the downloaded resource.
+        /// </summary>
+        Reload,
+
+        /// <summary>
+        /// The browser looks for a matching request in its HTTP cache.
```

Comment:
Looks like a wrongly copy/pasted description.

Reply:
I've copied the description verbatim from https://developer.mozilla.org/en-US/docs/Web/API/Request/cache